idd/Energy+.idd.in

```diff
@@ -42751,6 +42773,93 @@ Coil:Heating:DX:VariableRefrigerantFlow,
        \note cubic curve = a + b*ff + c*ff**2 + d*ff**3
        \note ff = fraction of the full load flow
 
+Coil:Cooling:DX:VariableRefrigerantFlow:FluidTemperatureControl, 
+       \memo This object models the indoor unit coil of the Variable Refrigerant Flow system (cooling mode)
+       \memo Used with ZoneHVAC:TerminalUnit:VariableRefrigerantFlow. 
+       \memo Outdoor unit is modeled separately, see AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl.
+       \min-fields 7
+  A1,  \field Name
+       \required-field
+       \type alpha
+       \reference CoolingCoilsDXVarRefrigFlowFluidTemperatureControl
+  A2,  \field Availability Schedule Name
+       \type object-list
+       \object-list ScheduleNames
+       \note Enter the name of a schedule that defines the availability of the coil
+       \note Schedule values of 0 denote the unit is off. All other values denote the unit is available
+       \note If this field is left blank, the unit is available the entire simulation
+  A3,  \field Coil Air Inlet Node
+       \required-field
+       \type node
+       \note the inlet node to the coil
+  A4,  \field Coil Air Outlet Node
+       \required-field
+       \type node
+       \note the outlet node to the coil
+  N1,  \field Rated Total Cooling Capacity
+       \required-field
+       \type real
+       \units W
+       \minimum> 0.0
+       \note Supply air fan heat is not included
+  N2,  \field Rated Sensible Heat Ratio
+       \required-field
+       \type real
+       \minimum> 0.0
+       \note Supply air fan heat is not included
+  N3,  \field Indoor Unit Reference Superheating 
+       \required-field
+       \type real
+       \units deltaC 
```

Comment:
Have you tried using autosize for Rated Total Cooling Capacity and Rated Sensible Heat Ratio? Is it possible to autosize these inputs? Can you add a default for Indoor Unit Reference Superheating since this value is typically around 5C? If you do add a default, remove the \required-field tag.